examples/README-CN.md

```diff
@@ -0,0 +1,43 @@
+# 示例
```

Comment:
Similar to the docs, let's create a `zh` sub directory under `examples` and put all the Chinese examples inside `zh`.